docs/Maintainers.md

```diff
@@ -0,0 +1,50 @@
+Maintainers
+===========
+
+This document describes the Maintainers Committee of Open Enclave. By our
+liberal contribution policy outlined in our
+[governance model](GovernanceModel.md), maintainers are committers that are
+trusted to grant new committer rights, and grant new membership into the
+Maintainers Committee.
+
+When making decisions, the Maintainers Committee uses a "consensus seeking"
```

Comment:
This process needs to be elaborated upon and it should be moved after the responsibilities of a maintainer are discussed to put the context of what decisions they should be making in context.

Reply:
Can you explain what part needs to be elaborated on?

Reply:
Is consensus being used as a synonym for unanimous agreement here? That's the implication given that the fallback is majority vote. It seems like a weirdly high initial bar.

Reply:
Here's the excerpt on the article I linked explaining why this bar/process works well:

> The TC uses a “consensus seeking” process for issues that are escalated to the TC. The group tries to find a resolution that has no open objections among TC members. If a consensus cannot be reached that has no objections then a majority wins vote is called. It is also expected that the majority of decisions made by the TC are via a consensus seeking process and that voting is only used as a last-resort.
The best solution tends to be the one everyone can agree to so you would think that consensus systems would be the norm. However, pure consensus systems incentivize obstructionism which we need to avoid.
In pure consensus everyone essentially has a veto. So, if I don’t want something to happen I’m in a strong position of power over everyone that wants something to happen. They have to convince me and I don’t have to convince anyone else of anything.
To avoid this we use a system called “consensus seeking” which has a long history outside of open source. It’s quite simple, just attempt to reach a consensus, if a consensus can’t be reached then call for a majority wins vote.
Just the fact that a vote is a possibility means that people can’t be obstructionists, whether someone favor a change or not, they have to convince their peers and if they aren’t willing to put in the work to convince their peers then they probably don’t involve themselves in that decision at all.
The way these incentives play out is pretty impressive. We started using this process in io.js and adopted it in Node.js when we merged into the foundation. In that entire time we’ve never actually had to call for a vote, just the fact that we could is enough to keep everyone working together to find a solution and move forward.